src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -108,52 +107,53 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
         />
       }
     >
-      <PageBlock title={'Change Request Details'} headerRight={<b>{changeRequest.status}</b>}>
-        <Grid container spacing={1}>
-          <Grid item xs={2}>
-            <Typography sx={{ maxWidth: '140px', fontWeight: 'bold' }}>Type: </Typography>
+      <Grid container rowGap={3}>
+        <Grid container columnSpacing={3}>
+          <Grid item xs={'auto'}>
+            <Typography sx={{ fontWeight: 'normal', fontSize: '21px' }}>
+              <b>WBS: </b>
+              <Link component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}`}>
+                {wbsPipe(changeRequest.wbsNum)} - {projectName}
+                {isProject(changeRequest.wbsNum) ? '' : ' - ' + changeRequest.wbsName}
+              </Link>
+            </Typography>
           </Grid>
-          <Grid item xs={10}>
-            {changeRequest.type}
+          <Grid item xs={'auto'}>
+            <Typography sx={{ fontWeight: 'normal', fontSize: '21px' }}>
+              <b>Submitter: </b>
+              {fullNamePipe(changeRequest.submitter)} on {datePipe(changeRequest.dateSubmitted)}
+            </Typography>
           </Grid>
-          <Grid item xs={2}>
-            <Typography sx={{ fontWeight: 'bold' }}>WBS #: </Typography>
-          </Grid>
-          <Grid item xs={10}>
-            <Link component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}`}>
-              {wbsPipe(changeRequest.wbsNum)} - {projectName}
-              {isProject(changeRequest.wbsNum) ? '' : ' - ' + changeRequest.wbsName}
-            </Link>
+        </Grid>
+        <Grid container rowSpacing={2} columnSpacing={2}>
+          <Grid item xs={12}>
+            {buildDetails(changeRequest)}
           </Grid>
-          <Grid item xs={3} md={2}>
-            <Typography sx={{ fontWeight: 'bold' }}>Submitted By: </Typography>
+          <Grid item xs={12}>
+            {buildProposedSolutions(changeRequest)}
           </Grid>
-          <Grid item xs={2}>
-            <Typography>{fullNamePipe(changeRequest.submitter)}</Typography>
+          <Grid item xs={5}>
```

Comment:
the screen is pretty packed if you make it small enough (e.g. mobile view), can you make it so review notes and implemented changes stack vertically if it gets small enough
<img width="960" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/0438e2a1-d9e3-4208-89f8-8eecea8f6a32">